yt_dlp/extractor/showuptv.py

```diff
@@ -0,0 +1,42 @@
+# coding: utf-8
+from .common import InfoExtractor
+from datetime import datetime
+
+
+class ShowupTvIE(InfoExtractor):
+    IE_NAME = 'showup.tv'
+    _VALID_URL = r"https?://(?:www\.)?showup\.tv/(?P<id>.+)"
+
+    _TESTS = [{
+        "url": "https://showup.tv/username",
+        "info_dict": {
+            "id": "username",
+            "ext": "mp4",
+            'is_live': True,
+        },
+        "skip": "Website content changes dynamically, depending which users are online and available for download/streaming",
```

Comment:
This test is wrong! make a test that actually passes at the time of writing. Then add `skip` since it will break in future